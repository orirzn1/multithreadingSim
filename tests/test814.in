L13
S6
O9
N7

T0
I@0x00000000
ADDI $6, $4, 0x4
STORE $5, $1, $7
SUBI $1, $5, 0x1
ADDI $5, $1, 0x4
ADD $3, $7, $1
STORE $3, $6, $5
ADD $7, $3, $2
ADD $0, $1, $4
SUB $6, $5, $3
HALT $0

T1
I@0x00000000
ADD $4, $3, $7
SUB $2, $4, $3
ADDI $1, $1, 0x2
SUBI $0, $2, 0x4
SUBI $4, $2, 0x4
ADD $2, $0, $0
SUB $7, $2, $6
ADD $0, $4, $2
SUBI $5, $5, 0x1
SUBI $3, $0, 0x4
HALT $0

T2
I@0x00000000
SUBI $6, $4, 0x3
SUB $1, $1, $0
SUBI $2, $2, 0x1
LOAD $2, $2, $7
ADD $7, $7, $5
SUB $0, $5, $0
ADD $7, $5, $3
STORE $1, $6, $7
LOAD $7, $4, 0x4
STORE $1, $7, 0x2
HALT $0

T3
I@0x00000000
ADDI $3, $3, 0x3
SUBI $7, $3, 0x4
ADDI $4, $0, 0x2
LOAD $7, $0, $4
STORE $1, $6, $1
STORE $2, $5, $7
HALT $0

T4
I@0x00000000
ADDI $2, $6, 0x4
ADD $2, $5, $5
ADDI $5, $2, 0x4
STORE $5, $1, 0x2
HALT $0

T5
I@0x00000000
LOAD $7, $5, $0
ADD $0, $6, $6
ADD $4, $7, $5
ADD $6, $7, $2
HALT $0

T6
I@0x00000000
SUB $7, $3, $4
LOAD $2, $0, $1
ADD $7, $5, $5
ADD $4, $2, $1
ADD $4, $1, $7
HALT $0

D@0x0
0x7
0x5
0x2
0xA
0x8
0x9
0x7
0x5
0x9
