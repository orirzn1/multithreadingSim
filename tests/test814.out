
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0xFFFFFFFF	R1 = 0xFFFFFFFF	R2 = 0x0	R3 = 0xFFFFFFFF	R4 = 0x0	R5 = 0x3	R6 = 0x4	R7 = 0xFFFFFFFF
Register file thread id 1:
	R0 = 0xFFFFFFF4	R1 = 0x2	R2 = 0xFFFFFFF8	R3 = 0xFFFFFFF0	R4 = 0xFFFFFFFC	R5 = 0xFFFFFFFF	R6 = 0x0	R7 = 0xFFFFFFF8
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFD	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x3	R4 = 0x2	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x4	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 6:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Blocked MT CPI for this program 4.000000

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0xFFFFFFFF	R1 = 0xFFFFFFFF	R2 = 0x0	R3 = 0xFFFFFFFF	R4 = 0x0	R5 = 0x3	R6 = 0x4	R7 = 0xFFFFFFFF
Register file thread id 1:
	R0 = 0xFFFFFFF4	R1 = 0x2	R2 = 0xFFFFFFF8	R3 = 0xFFFFFFF0	R4 = 0xFFFFFFFC	R5 = 0xFFFFFFFF	R6 = 0x0	R7 = 0xFFFFFFF8
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFD	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x3	R4 = 0x2	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x4	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 6:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Finegrained Multithreading CPI for this program 1.381818

